src/NodeVisitor/RemovedFunctionCallVisitor.php

```diff
@@ -8,18 +8,39 @@
 class RemovedFunctionCallVisitor extends AbstractVisitor
 {
     protected $removedFunctionNames = array(
+        // Removed in favor of call_user_func* functions
         'call_user_method',
-        'call_user_method',
-        'dl',
```

Comment:
dl() can no longer be used in PHP-FPM but it remains functional in the CLI and embed SAPIs so I removed it from the list